vars/artifactsPipeline.groovy

```diff
@@ -0,0 +1,152 @@
+#! groovy
+
+def call(Map pipelineParams) {
+    pipeline {
+        agent {
+            label {
+                label getJenkinsLabels('gce')
+            }
+        }
+        environment {
+            AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
+            AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
+        }
+        parameters {
+            string(defaultValue: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-3.2.repo',
```

Comment:
We can't set this by default.

Reply:
Please, can you explain why?

Reply:
The value should be transferred from the job that build the rpms.
It's not safe here to put a default.

Reply:
done